R/discover_rules.R

```diff
@@ -1,23 +1,24 @@
 #' @title
-#' Generate causal rules
+#' Discover Rules
 #'
 #' @description
-#' Generates causal decision rules.
+#' Discover the minimal set of Rules Linearly Decomposing the Conditional
```

Comment:
`Rules Linearly Decomposing` please change to small case letters.

Reply:
addressed in the new PR